wgpu/src/lib.rs

```diff
@@ -1277,16 +1277,24 @@ impl Display for SurfaceError {
 
 impl error::Error for SurfaceError {}
 
+impl Default for Instance {
+    /// Creates a new instance of wgpu with default options.
+    /// Backends are set to `Backends::all()`, and FXC is chosen as the `dx12_shader_compiler`.
```

Comment:
```suggestion
    /// Creates a new instance of wgpu with default options.
    ///
    /// Backends are set to `Backends::all()`, and FXC is chosen as the `dx12_shader_compiler`.
```

The break will make the first line the summary shown on the module page